sympy/sets/tests/test_sets.py

```diff
@@ -941,3 +942,13 @@ def test_issue_10113():
 def test_issue_10248():
     assert list(Intersection(S.Reals, FiniteSet(x))) == [
         And(x < oo, x > -oo)]
+
+
+def test_set_in_interval():
+    x = Symbol('x', real=True)
+    assert x in Interval(x, x+4)
+    assert Interval(1, 2) not in Interval(0, 5)
```

Comment:
> "in" means element of, not subset of.

OK, so thinking of a set of items, I can see that for preciseness reasons we would want `Interval(2,3) in FiniteSet(Interval(1,4), foo)` to return False -- it's literal in the sense that *that* Interval is not in the set just as `'a' in ['abc', 'foo']` would be false while `any('a' in I for I in ['abc', 'foo'])` would be true.